app/models/package_manager/pypi.rb

```diff
@@ -135,7 +135,7 @@ def self.parse_pep_508_dep_spec(dep)
 
     def self.dependencies(name, version, _mapped_project = nil)
       api_response = get("https://pypi.org/pypi/#{name}/#{version}/json")
-      deps = api_response.dig("info", "requires_dist")
+      deps = api_response.dig("info", "requires_dist") || []
```

Comment:
This just prevents us from logging if `requires_dist` is `null`. It was raising on the `deps.map` below, but it seems like it's Pypi's way (or _a way_) of indicating that there are no dependencies.